app/src/androidTest/java/org/mozilla/focus/activity/SwitchContextTest.java

```diff
@@ -112,7 +112,7 @@ public void ForegroundTest() throws InterruptedException, UiObjectNotFoundExcept
 
         // Assert website is loaded
         assertTrue(TestHelper.webView.waitForExists(waitingTime));
-        Assert.assertTrue("Website title loaded", titleMsg.exists());
+        Assert.assertTrue("Website title loaded", titleMsg.waitForExists(waitingTime));
```

Comment:
\[PMD\] INFO: The String literal "Website title loaded" appears 5 times in this file; the first occurrence is on line 115